src/Native/Unix/CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ if (NOT WIN32)
         # Ensure that objcopy is present
         if(DEFINED ENV{CROSSCOMPILE})
             if(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l OR CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL i686)
-                find_program(OBJCOPY ${TOOLCHAIN}-objcopy)
+                find_program(OBJCOPY ${TOOLCHAIN_PREFIX}objcopy)
```

Comment:
I'm not sure why the scripts differ; I expect this change simply wasn't propagated from coreclr.  It'd be great to make them as close as possible.  @leemgs, any reason for this difference?